test-data/unit/stubgen.test

```diff
@@ -168,16 +203,22 @@ x, y = 1, 2
 [out]
 from typing import Any
 
-x = ...  # type: Any
-y = ...  # type: Any
+x: Any
+y: Any
+
+[case testMultipleAssignmentAnnotated]
+x, y = 1, "2" # type: int, str
```

Comment:
OK... the current code is not crashing, but generating "something" instead of erroring out. I can put an error message although I'm a bit converned that validation is a bit out of scope from the stubgen tool.